### PR TITLE
Add debug logs for ConsumerGroup reconciler to record actions taken

### PR DESCRIPTION
**Pull request by pierDipi**
Helps with debugging

### CODE REVIEW

File: control-plane/pkg/reconciler/consumergroup/consumergroup.go [hunk 1]
```diff
@@ -185,20 +185,27 @@ type Reconciler struct {
 }
 
 func (r *Reconciler) ReconcileKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
+	logger := logging.FromContext(ctx)
```
Comment (creydr) [inline control-plane/pkg/reconciler/consumergroup/consumergroup.go:188]: nit: could we add the name&namespace of the consumergroup, so we can see it directly from the logs?

**Review by creydr — COMMENTED** [review]
/lgtm

/hold
in case you want to address the nit

File: control-plane/pkg/reconciler/consumergroup/consumergroup.go (hunk 1 above)
Comment (pierDipi) [reply control-plane/pkg/reconciler/consumergroup/consumergroup.go:188]: It's already there

Comment (pierDipi) [reply control-plane/pkg/reconciler/consumergroup/consumergroup.go:188]: injected loggers are pre-configured with a few standard keys, see knative/pkg for details

**Comment by Cali0707** [comment]
Thanks @pierDipi !!

**Pull request merged**